### PR TITLE
build: update windows toolchain

### PR DESCRIPTION
**Pull request by kyessenov**
Commit Message: bump CI image
Additional Description:
Risk Level:
Testing:
Docs Changes:
Release Notes:

### CODE REVIEW

File: .github/workflows/env.yml [hunk 1]
```diff
@@ -75,9 +75,9 @@ jobs:
       env:
         # TODO(phlax): derive these from a config file
         BUILD_IMAGE_UBUNTU_REPO: envoyproxy/envoy-build-ubuntu
-        BUILD_IMAGE_UBUNTU: 41c5a05d708972d703661b702a63ef5060125c33
-        BUILD_IMAGE_UBUNTU_SHA: 50337314a150ed12447c87c1622eac6f611a069888722fb9a426e21ed161cc26
-        BUILD_IMAGE_UBUNTU_MOBILE_SHA: ca26ff05bd3f3a09468242faaf38ae48315e57f0a87c102352162f95ac620e6f
+        BUILD_IMAGE_UBUNTU: 105fdc54acc55ff0ff0c5b784d8cfe8dc225aad4
+        BUILD_IMAGE_UBUNTU_SHA: 1e89eb7136b4d53b021c52b3497255e6221da8a2003d7c727e397e4ecb227d9b
+        BUILD_IMAGE_UBUNTU_MOBILE_SHA: 1e89eb7136b4d53b021c52b3497255e6221da8a2003d7c727e397e4ecb227d9b
```
Comment (phlax) [inline .github/workflows/env.yml:80]: you can get this one with ...

```console
$ docker buildx imagetools inspect envoyproxy/envoy-build-ubuntu:mobile-105fdc54acc55ff0ff0c5b784d8cfe8dc225aad4
```

File: mobile/third_party/rbe_configs/config/BUILD [hunk 2]
```diff
@@ -43,8 +43,8 @@ platform(
     ],
     exec_properties = {
         # Please update both the commented tag and the sha256
-        # mobile-41c5a05d708972d703661b702a63ef5060125c33
-        "container-image": "docker://envoyproxy/envoy-build-ubuntu@sha256:ca26ff05bd3f3a09468242faaf38ae48315e57f0a87c102352162f95ac620e6f",
+        # mobile-105fdc54acc55ff0ff0c5b784d8cfe8dc225aad4
+        "container-image": "docker://envoyproxy/envoy-build-ubuntu@sha256:1e89eb7136b4d53b021c52b3497255e6221da8a2003d7c727e397e4ecb227d9b",
```
Comment (phlax) [inline mobile/third_party/rbe_configs/config/BUILD:47]: these 2 will need the output from above command also

Comment (phlax) [reply mobile/third_party/rbe_configs/config/BUILD:47]:
```suggestion
        "container-image": "docker://envoyproxy/envoy-build-ubuntu@sha256:6dcedeb63b6dcc74d5991af858638a711d6b15a37466349c48373f682bfda8c1",
```

File: mobile/third_party/rbe_configs/config/BUILD [hunk 3]
```diff
@@ -60,8 +60,8 @@ platform(
...
```
Comment (phlax) [inline mobile/third_party/rbe_configs/config/BUILD:64]:
```suggestion
        "container-image": "docker://envoyproxy/envoy-build-ubuntu@sha256:6dcedeb63b6dcc74d5991af858638a711d6b15a37466349c48373f682bfda8c1",
```

**Review by phlax — COMMENTED** [review]
its not clear if the windows fail is real - seems container related but may be just pulling a new image or somesuch

**Comment by kyessenov** [comment]
Looks like a genuine failure with absl/synchronization `failure in a Windows system call: The system cannot find the file specified`. I can't proceed further since I need a windows dev enviornment to debug.

**Comment by phlax** [comment]
>  I can't proceed further since I need a windows dev enviornment to debug.

yeah, this is a recurring issue

probs we should back out the upstream change until we have resolved

cc @davinci26 @wrowe @sunjayBhatia

**Comment by sunjayBhatia** [comment]
sorry, not really working too much on Windows at the moment so i don't have a workstation handy but this might be relevant to the attempted bump: https://learn.microsoft.com/en-us/visualstudio/releases/2022/system-requirements#microsoft-visual-studio-build-tools-2022-system-requirements

**Comment by kyessenov** [comment]
I see. @phlax we can revert my change until we can test run the container somehow, since the container didn't build correctly.

**Pull request closed without merging**